presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleTester.java

```diff
@@ -83,7 +84,9 @@ public RuleTester(List<Plugin> plugins, Map<String, String> sessionProperties, O
 
     public RuleTester(List<Plugin> plugins, Map<String, String> sessionProperties, Optional<Integer> nodeCountForStats, ConnectorFactory connectorFactory)
     {
-        this(plugins, sessionProperties, new SessionPropertyManager(), nodeCountForStats, connectorFactory);
+        this(plugins, sessionProperties,
```

Comment:
formatting nit.  If we don't have everything on one line, we do one argument per line with the first argument on the line after the open parentheses. as follows:
```
myMethod(
      arg1,
      arg2,
      arg3);
```